docs/reference/ml/df-analytics/apis/put-dfanalytics.asciidoc

```diff
@@ -62,14 +66,20 @@ and mappings.
   index. For example: `outlier_detection`. See <<dfanalytics-types>>.
   
 `analyzed_fields`::
-  (Optional, object) You can specify both `includes` and/or `excludes` patterns. If 
-  `analyzed_fields` is not set, only the relevant fields will be included. For 
-  example, all the numeric fields for {oldetection}.
+  (Optional, object) You can specify both `includes` and/or `excludes` patterns. 
+  If `analyzed_fields` is not set, only the relevant fields will be included. 
+  For example, all the numeric fields for {oldetection}.
   
 `dest`::
   (Required, object) The destination configuration, consisting of `index` and 
-  optionally `results_field` (`ml` by default). See 
-  <<ml-dfanalytics-properties,{dfanalytics} properties>>.
+  optionally `results_field` (`ml` by default).
+  
+    `index`:::
+      (Required, string) The _destination index_ for the {dfanalytics}.
```

Comment:
The definition list format does not seem to respect indentation, and the list items are presented at the same level as their parent. 
<img width="681" alt="Screen Shot 2019-08-16 at 9 12 31 PM" src="https://user-images.githubusercontent.com/35154725/63204900-d4c11200-c06a-11e9-8674-b7861c9e8f3d.png">

You have three colons on line 77, and that one appears to indent correctly.  I tested that approach with the other values, and they lined up correctly, too.  I couldn't find any formal documentation that supports this approach, but it seems to work at least on my local build.  I'll retest after you incorporate the other review comments.

Reply:
@karenzone Thank you for this, I went with three colons. According to my local build and the Jenkins test, this change solves the indentation issue.